simexpal/base.py

```diff
@@ -165,6 +171,11 @@ def construct_variants():
 					raise RuntimeError("The revision name '{}' is ambiguous".format(revision.name))
 				self._revisions[revision.name] = revision
 
+		for axis in sorted(construct_axes(), key=lambda axis: axis):
```

Comment:
`key=lambda axis: axis` is the default - why write this explicitly?

Reply:
I wrote it explicitly to keep that code consistent with the other parts and with the idea of adding an `Axis` class that could f.e. have references to its variants, which I didn't do but then left that code as is. I can remove that redundancy - should I also change `Config._axes` to a list, rather than a dict?